test/Elastic.Apm.AspNetFullFramework.Tests/Elastic.Apm.AspNetFullFramework.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
I'm leaning towards targeting net4x. Is there any other advantages to targeting .NET Core other than having one less project fail to build on non Windows OS? I've created #306 to improve this aspect of development experience on non Windows OS.

Reply:
> Is there any other advantages to targeting .NET Core other than having one less project fail to build on non Windows OS?

I think that's the only advantage, plus, `net4x` probably won't have any new features, since that's not the focus of Microsoft - which I'd say isn't a problem for this test project.

I'm ok with targeting `net4x`. Also, since we have #306, I think it's also fine to merge it as it is, and in #306 we can come up with a general solution that addresses both this project and the other full framework projects. Choose whichever is easier for you.

Reply:
I tried to change `Elastic.Apm.AspNetFullFramework.Tests` to target net461 but it fails to build because it depends on `Elastic.Apm.Tests.MockApmServer` which targets `netcoreapp2.1`. So I would prefer to keep `Elastic.Apm.AspNetFullFramework.Tests` targeting `netcoreapp2.1`. WDTY?

Reply:
That's fine, definitely not a blocker, will be ok in CI and I can deal with it in my local dev. env. Let's see what we can do in  #306...

Reply:
> I tried to change `Elastic.Apm.AspNetFullFramework.Tests` to target net461 but it fails to build because it depends on `Elastic.Apm.Tests.MockApmServer` which targets `netcoreapp2.1`. So I would prefer to keep `Elastic.Apm.AspNetFullFramework.Tests` targeting `netcoreapp2.1`. WDTY?

This is a bad idea, and your attempt to change `Elastic.Apm.AspNetFullFramework.Tests` to target `net461` was the correct approach. I've raised issue #315 regarding this and put some suggestions in there on how to resolve it.